app/src/main/java/ch/epfl/polychef/pages/HomePage.java

```diff
@@ -80,29 +77,38 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    @Override
-    public void onStart() {
-        super.onStart();
+    public void setupDrawer(){
```

Comment:
Could we set this method private ? (same for setupUserInfo, setupProfilePicture, setupProfileNavigation)

Reply:
Yes, except for setupProfilePicture, since we need to be able to change profile pictures. I'll change that